shared/middleware/helpers/core.rb

```diff
@@ -46,9 +46,19 @@ def not_found()
 end
 
 def too_large()
-  halt(413, "Payload too large\n")
```

Comment:
Yep, this is true. My question was mostly around whether I should try to make helpers that work everywhere, or just create a second set of helpers that I use when not in a class inheriting from Sinatra::Base